### PR TITLE
fix:addressed cve suppressions(RDCC-1740)

### PR DESCRIPTION
**Pull request by abhijitdiwan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-1742

### Change description ###
fix:addressed cve suppressions(RDCC-1740)

**Does this PR introduce a breaking change?** (check one with "x")
No

### CODE REVIEW

**Review by shreedhar-hmcts — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -311,13 +311,12 @@ dependencies {
 
     compile group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
     compile group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
-    compile group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version: '0.0.4'
+    compile group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version: '0.1.0'
     compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: versions.reformS2sClient
     compile group: 'uk.gov.hmcts.reform', name: 'health-spring-boot-starter', version: versions.reformHealthStarter
-    compile(group: 'uk.gov.hmcts.reform.auth', name: 'auth-checker-lib', version: '2.1.3') {
+    compile(group: 'uk.gov.hmcts.reform.auth', name: 'auth-checker-lib', version: '2.1.4') {
         // TODO remove when auth-checker-lib upgrades its dependency to spring boot 2.0.x
```
Comment (KOTLAPRA23) [inline build.gradle:315]: we  don't need auth-checker-lib. you can remove it

**Review by shashank-rastogi — APPROVED** [review]

**Review by KOTLAPRA23 — COMMENTED** [review]
it's good

File: build.gradle (hunk 1 above)
Comment (abhijitdiwan) [reply build.gradle:315]: Yes we can remove it. Lets keep for now and we will aggregate all and remove.

Comment (shashank-rastogi) [reply build.gradle:315]: remove it now... as ingestion lib does not need it

**Pull request merged**